VideoAPI/VideoApi.DAL/Commands/AssignLeaveTimeForRoomParticipantCommand.cs

```diff
@@ -0,0 +1,53 @@
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Linq;
+using VideoApi.DAL.Commands.Core;
+using VideoApi.DAL.Exceptions;
+using Task = System.Threading.Tasks.Task;
+
+namespace VideoApi.DAL.Commands
+{
+    public class AssignLeaveTimeForRoomParticipantCommand : ICommand
```

Comment:
The leave time being assigned is a consequence for a participant leaving the consultation room. I think  `RemoveParticipantFromConsultationFroom' would be more representative of the expected behaviour

Reply:
typo in `RemoveParticipantFromConsultationFroom` @shaed-parkar  :P ;)

Reply:
The command to assign leave time is removed, we don't keep history